backend/app/usecase/url/urlretriever.go

```diff
@@ -55,9 +57,21 @@ func (r RetrieverPersist) getURL(alias string) (entity.URL, error) {
 	return url, nil
 }
 
+// GetURLsByUser retrieves URLs from persistent storage created by given user
+func (r RetrieverPersist) GetURLsByUser(user entity.User) ([]entity.URL, error) {
+	aliases, err := r.userURLRelationRepo.FindAliasesByUser(user)
```

Comment:
I think we should change  `user entity.User` to `user *entity.User` and handle `nil` case here to decouple business logics from APIs.